File: core/src/org/tetawex/ecf/util/JsonUtils.java
package org.tetawex.ecf.util;

import com.google.gson.Gson;

/**
 * Created by tetawex on 16.08.17.
 */
public class JsonUtils {
    private static Gson gson;

    public static Gson getSerializer() {
        if (gson == null)
            gson = new Gson();
        return gson;
    }
}

File: core/src/org/tetawex/ecf/util/JsonUtils.kt
package org.tetawex.ecf.util

import com.google.gson.Gson

/**
 * Created by tetawex on 16.08.17.
 */
object JsonUtils {
    private var gson: Gson? = null

    val serializer: Gson
        get() {
            if (gson == null)
                gson = Gson()
            return gson!!
        }
}
